lib/driver-helper.js

```diff
@@ -45,6 +45,55 @@ export function clickWhenClickable( driver, selector, waitOverride ) {
 	);
 }
 
+export function clickWhenFocused( driver, selector, pollingOverride, waitOverride ) {
```

Comment:
As per my comment here https://github.com/Automattic/wp-e2e-tests/pull/1698#discussion_r240554182 the reason why the tests that run too fast are failing could be that there is no check if the cursor had been placed in the input field or if the input field has a focus. The input field may be disaplyed but not in focus yet while the driver is trying to clear it already.

To address this issue I wrote a new `driverHelper` method `clickWhenFocused`. It is based on the following steps:

- Find element;
- Make it active (in focus) by switching to it;
- Make sure that this element equals the element on which the next action must be performed;
- Click on that element.

The focus is implemented by using `switchTo().activeElement()`. 

`activeElement()` switches to the element that currently has focus within the document currently "switched to", or the body element if this cannot be detected. See reference [here](https://seleniumhq.github.io/selenium/docs/api/java/org/openqa/selenium/WebDriver.TargetLocator.html#activeElement--). 

Once the element is in focus we need to make sure that it matches the element that needs to be clicked on. To do that, we can utilize `getId()` which is a server assigned `Id` to the element of the DOM of the particualr session (not the HTML `Id`) as per reference [here](https://github.com/SeleniumHQ/selenium/blob/master/javascript/node/selenium-webdriver/lib/webdriver.js#L1829).  

@bsessions85 let me know your thoughts. Also cc @Stojdza as you wanted to work on re-writing `clickWhenClickable` with a similar approach.

Reply:
@juliaamosova This method is also applicable to buttons and other elements? If yes, did you consider to use it as `clickWhenClickable` method (rewrite existing one)? Those two look similar, and it's easier just to change the existing one and use it through all tests.

Reply:
@Stojdza I did consider re-writing `clickWhenClickable` but didn't want to use it before the review of the new method I wrote. I agree that keeping `clickWhenClickable` will be better. The method I wrote should be applicable to all types of elements. Let me test it locally and I will get back with a new PR.

Reply:
@Stojdza I refactored `clickWhenClickable` with the changes I introduced in my `clickWhenFocused`method. While it is working locally, I see that some tests are failing here mainly due to timeouts. I am looking to see what could be going on.

Reply:
@Stojdza after additional testing, I can't confirm that I can re-write `clickWhenClickable` with changes I introduced in `waitTillFocused` method. It seems to work in many cases but not always. I removed the `click` event from the  `waitTillFocused` so that it could be used to make sure the element is in focus only. It works great to address the sleep statements in `login-page.js`.